pkg/cmd/pulumi/deployment_settings_config.go

```diff
@@ -422,7 +470,8 @@ func configureGit(ctx context.Context, displayOpts *display.Options, be backend.
 		}
 	}
 
-	repoDir, err := cmdutil.ReadConsoleWithDefault("Enter the repo directory", defaultRepoDir)
+	repoDir, err := promptForValue(d.Yes, "repo directory", defaultRepoDir, false,
```

Comment:
Not sure whether we consistently shorten or spell out things in full, but I feel like "repository directory" (or "path to the repository", etc.) would be better. I see we use "repository" elsewhere in this PR, so I think we should be consistent.

Reply:
done